app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -333,11 +332,17 @@ class NavigationDrawerTestActivityTest {
 
   @Test
   // TODO(#973): Fix NavigationDrawerTestActivityTest
-  @Ignore
   fun testNavigationDrawerTestActivity_withAdminProfile_openNavigationDrawer_checkAdministratorControlsDisplayed() { // ktlint-disable max-line-length
-    launch<NavigationDrawerTestActivity>(createNavigationDrawerActivityIntent(0)).use {
-      onView(withContentDescription(R.string.drawer_open_content_description)).perform(click())
-      onView(withId(R.id.administrator_controls_linear_layout)).check(matches(isDisplayed()))
+    launch(NavigationDrawerTestActivity::class.java).use {
```

Comment:
Could you describe a bit about this change here!!

as per the earlier test, we are launching `NavigationDrawerTestActivity` using `createNavigationDrawerActivityIntent` where we are passing `0` as to mention the Admin profile, but how this new change is different from the previous one?